core/DataAccess/RawLogUpdater.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\DataAccess;
+
+use Piwik\Common;
+use Piwik\Db;
+
+class RawLogUpdater
```

Comment:
That was marcin's decision, not sure what the reason behind it was. We could merge them together. Would you have an issue w/ the name `RawLogDao` or `RawLogDataDao` or `RawLog(Data)Model`?